tests/common.go

```diff
@@ -5398,6 +5398,62 @@ func CreateS3Bucket(bucketName string, objectLock bool, retainCount int64, objec
 	return err
 }
 
+// UpdateS3BucketPolicy applies the given policy to the given bucket.
+func UpdateS3BucketPolicy(bucketName string, policy string) error {
+
+	id, secret, endpoint, s3Region, disableSSLBool := s3utils.GetAWSDetailsFromEnv()
+	sess, err := session.NewSession(&aws.Config{
+		Endpoint:         aws.String(endpoint),
+		Credentials:      credentials.NewStaticCredentials(id, secret, ""),
+		Region:           aws.String(s3Region),
+		DisableSSL:       aws.Bool(disableSSLBool),
+		S3ForcePathStyle: aws.Bool(true),
+	},
+	)
+	expect(err).NotTo(haveOccurred(),
```

Comment:
Change this to log.FailOnError and everywhere else

Reply:
done

Reply:
done